shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Serialization.hs

```diff
@@ -502,395 +511,340 @@ roundTripIpv6 =
 serializationTests :: TestTree
 serializationTests =
   testGroup
-    "CBOR Serialization Tests"
-    [ checkEncodingCBOR
-        "list"
-        [1 :: Integer]
-        (T (TkListBegin . TkInteger 1 . TkBreak)),
-      checkEncodingCBOR
-        "set"
-        (Set.singleton (1 :: Integer))
-        (T (TkTag 258 . TkListLen 1 . TkInteger 1)),
-      checkEncodingCBOR
-        "map"
-        (Map.singleton (1 :: Integer) (1 :: Integer))
-        (T (TkMapLen 1 . TkInteger 1 . TkInteger 1)),
-      checkEncodingCBOR
-        "coin"
-        (Coin 30)
-        (T (TkWord64 30)),
-      testCase "prop_serialize_negative-coin" testNegativeCoin,
-      checkEncodingCBOR
-        "rational"
-        (UnsafeUnitInterval (1 % 2))
-        (T (TkTag 30 . TkListLen 2 . TkInteger 1 . TkInteger 2)),
-      checkEncodingCBOR
-        "slot"
-        (SlotNo 7)
-        (T (TkWord64 7)),
-      checkEncodingCBOR
-        "neutral_nonce"
-        NeutralNonce
-        (T (TkListLen 1 . TkWord 0)),
-      checkEncodingCBOR
-        "nonce"
-        (mkNonce 99)
-        (T (TkListLen 2 . TkWord 1 . TkBytes (getRawNonce $ mkNonce 99))),
-      checkEncodingCBOR
-        "key_hash"
-        testKeyHash1
-        (T (TkBytes (getRawKeyHash testKeyHash1))),
-      checkEncodingCBOR
-        "credential_key_hash"
-        testPayCred
-        (T (TkListLen 2 . TkWord 0) <> S testKeyHash1),
-      checkEncodingCBOR
-        "txin"
-        (TxIn genesisId 0 :: TxIn)
-        (T (TkListLen 2) <> S (genesisId :: TxId) <> T (TkWord64 0)),
-      let a = Addr Testnet testPayCred StakeRefNull
-       in checkEncodingCBOR
-            "txout"
-            (TxOut a (Coin 2))
-            ( T (TkListLen 2)
-                <> S a
-                <> S (Coin 2)
+    "Shelley Serialization Tests"
+    [ testGroup
+        "Serialization Property Tests"
+        [ testGroup
+            "Serialisation roundtrips"
```

Comment:
Do you plan on adding another `testGroup`? If not, I would remove the redundant `testGroup`.